src/tactic/interactive_expr.lean

```diff
@@ -153,6 +161,12 @@ meta def view {γ} (tooltip_component : tc subexpr (action γ)) (click_address :
     on_click (λ _, action.on_click ca),
     key s
   ] [html.of_string s]]
+| ca (sf.block i a) := do
+  inner ← view ca a,
+  pure [h "span" [cn "indent-code", style [("--indent-level", to_string i)]] inner]
```

Comment:
please could you add the CSS for `indent-code` to the PR description for context? Thanks

Reply:
done.